src/HumanView.cpp

```diff
@@ -23,53 +30,204 @@ void HumanView::init() {
+    sf::Sprite girl(girlText);
+    girl.setPosition(width - 400, 0);
+    sf::RectangleShape textBox(sf::Vector2f(650, 300));
+    textBox.setPosition(width / 9, height / 2);
+    textBox.setOutlineColor(sf::Color::Red);
+    sf::Text instruct(strings.getString("PressEnter"), font, 20);
+    instruct.setPosition(width / 5, 250);
+    instruct.setFillColor(sf::Color::White);
+    display.draw(girl);
+    display.draw(chad);
+    display.draw(textBox);
+    display.draw(instruct);
+}
+
+void HumanView::drawLevelDialogue() {
+    if (first) {
+        switch (dialogueStage) {
+            //eventually change to be appropriate color based on 
+            case 0: preference = strings.getPreference("Kill", 0); break;
+            case 1: preference = strings.getPreference("Ignore", 1); break;
+            case 2: preference = strings.getPreference("Kill", 2); break;
+        }
+        first = false;
+    }
+    if (dialogueStage > 3) {
+        return;
+    }
+    drawChadGirlBox();
+    sf::Text dialogue;
+    dialogue.setFont(font);  
+    dialogue.setString(preference);
+    dialogue.setFillColor(sf::Color::Magenta);
+    dialogue.setPosition(width / 8, height / 2);
+    display.draw(dialogue);
+    display.display();
+}
+
+void HumanView::drawEndLevelDialogue() {
+    if (first) {
+        switch (dialogueStage) {
+            //eventually change to be appropriate based on score
+            case 0: response = strings.getResponse("DateYes"); break;
+            case 1: response = strings.getResponse("Rejections"); break;
+        }
+        first = false;
+    }
+    view.setCenter(width / 2, height / 2);
+    display.setView(view);
+    if (dialogueStage > 2) {
+        return;
+    }
+    drawChadGirlBox();
+    sf::Text dialogue;
+    dialogue.setFont(font);
+    dialogue.setString(response);
+    dialogue.setFillColor(sf::Color::Red);
+    dialogue.setPosition(width / 8, height / 2);
+    display.draw(dialogue);
+    display.display();
+
+}
+
+void HumanView::checkKeyboardSettings() {
+    if (sf::Keyboard::isKeyPressed(sf::Keyboard::Backspace)) {
+        //go back to main menu
+        logic.setGameState(0);
+    }
+    
+}
+
+void HumanView::checkKeyPressed(sf::Keyboard::Key key) {
+    if (logic.getGameState() == 3 && waitingForKey == 0) {
+        if (key == sf::Keyboard::Right) {
+            waitingForKey = 1;
+        }
+        if (key == sf::Keyboard::Left) {
+            waitingForKey = 2;
+        }
+        if (key == sf::Keyboard::Up) {
+            waitingForKey = 3;
+        }
+        if (key == sf::Keyboard::Space) {
+            waitingForKey = 4;
+        }
+    }
+    else if (waitingForKey > 0) {
+        switch (waitingForKey) {
+            case 1: right = key; break;
+            case 2: left = key; break;
+            case 3: up = key; break;
+            case 4: shoot = key; break;
+        }
+        waitingForKey = 0;
+    }
+}
+
+void HumanView::checkKeyboardDialogue(float time) {
```

Comment:
Leave a comment to explain the gist of what each of these checkKeyboard functions do